### ISSUE TITLE
Cannot install AADInternals on Mac because Bitdefender finds virus

### ISSUE THREAD

**Issue by markb-trustifi** [Gerenios/AADInternals#90]
I receive this error when trying to install `AADInternals` on Mac v12.7.4:
`Install-Package: Package 'AADInternals' failed to be installed because: Access to the path '/tmp/f0dt1nhy/CommonUtils.ps1' is denied.`
Bitdefender message:
```
Virus deleted
Name: Heur.BZC.PZQ.Boxter.971.7A619...
FileName: CommonUtils.ps1
```

**Comment by michaelmsonne** [comment]
Yes, not many providers like this PowerShell Module - but it's with the stuff it's doing and so 😎

You can writelist it, as I not think (and what I have seen over the years..) it will be "allowed"/writelisted for the vendors.

**Comment by NestoriSyynimaa** [comment]
https://x.com/DrAzureAD/status/1838527114632401382

**Comment by NestoriSyynimaa** [comment]
After v0.9.6 release, this should not happen with AADInternals module anymore.

https://aadinternals.com/post/aadinternals-endpoints/
